Sources/Validated.swift

```diff
@@ -1,3 +1,8 @@
+#if swift(>=3.0)
+#else
+    public typealias ErrorProtocol = ErrorType
+#endif
+
```

Comment:
It gets mad at me when I try to do that 😄  I think it has to be written this way unfortunately.